build/Nuget/Microsoft.VisualStudio.ProjectSystem.nuget.proj

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I think we should call this package something else, this isn't the nuget package for the project system, this the nuget package for *insertion* into VS.